ai/mistralai/v0/config/tasks.json

```diff
@@ -0,0 +1,429 @@
+      "title": "Usage",
+      "type": "object"
+    },
+    "embedding-usage": {
+      "description": "Token usage on the Mistral platform embedding models",
+      "instillUIOrder": 1,
+      "properties": {
+        "tokens": {
+          "description": "The token count used by Mistral models",
+          "instillFormat": "number",
+          "instillUIOrder": 1,
+          "title": "Token Count",
+          "type": "number"
+        }
+      },
+      "required": [
+        "tokens"
+      ],
+      "title": "Usage",
+      "type": "object"
+    }
+  },
+  "TASK_TEXT_GENERATION_CHAT": {
+    "instillShortDescription": "Provide text outputs in response to text inputs.",
+    "description": "Mistral AI's text generation models (often called generative pre-trained transformers or large language models) have been trained to understand natural language, code, and images. The models provide text outputs in response to their inputs. The inputs to these models are also referred to as \"prompts\". Designing a prompt is essentially how you \u201cprogram\u201d a large language model model, usually by providing instructions or some examples of how to successfully complete a task",
+    "input": {
+      "description": "Input",
+      "instillEditOnNodeFields": [
+        "prompt",
+        "model-name"
+      ],
+      "instillUIOrder": 0,
+      "properties": {
+        "chat-history": {
+          "description": "Incorporate external chat history, specifically previous messages within the conversation. Please note that System Message will be ignored and will not have any effect when this field is populated. Each message should adhere to the format: : {\"role\": \"The message role, i.e. 'system', 'user' or 'assistant'\", \"content\": \"message content\"}",
+          "instillAcceptFormats": [
+            "structured/chat-messages"
+          ],
+          "instillShortDescription": "Incorporate external chat history, specifically previous messages within the conversation. (Note: The Mistral models are not trained to process images, thus images will be omitted)",
+          "instillUIOrder": 4,
+          "instillUpstreamTypes": [
+            "reference"
+          ],
+          "items": {
+            "$ref": "#/$defs/chat-message"
+          },
+          "title": "Chat history",
+          "type": "array"
+        },
+        "max-new-tokens": {
+          "default": 50,
+          "description": "The maximum number of tokens for model to generate",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUIOrder": 6,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Max new tokens",
+          "type": "integer"
+        },
+        "model-name": {
+          "enum": [
+            "open-mixtral-8x22b",
+            "open-mixtral-8x7b",
+            "open-mistral-7b",
+            "mistral-large-latest",
+            "mistral-small-latest"
```

Comment:
Question
How about this model?
Though it seems not Chat, the use way is same as the chat one. right?

<img width="853" alt="image" src="https://github.com/instill-ai/component/assets/64747455/023dfc0e-0a4e-4562-9216-1dce86976ab5">

Reply:
Added. This model could be use in both chat completion and Fill-In-Middle completion, however, the FIM completion is less suited for our use case. So I don't think we have to support it right now.